docs/error-messages/compiler-errors-1/compiler-error-c2323.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Compiler Error C2323"
+description: "Learn more about: Compiler Error C2323"
+ms.date: "03/20/2024"
+f1_keywords: ["C2323"]
+helpviewer_keywords: ["C2323"]
+---
+# Compiler Error C2323
+
+'identifier': non-member operator `new` or `delete` functions may not be declared `static` or in a namespace other than the global namespace.
```

Comment:
Should keywords in error messages have backticks or be shown verbatim (raw with no backticks)? Since what I commonly see is the latter.

Reply:
The goal is to prevent machine translation. We weren't good about this and got complaints about crazy machine translations for things that shouldn't be translated in the first place. So, we started adding the back ticks.